non-critical-infra/modules/mailserver/default.nix

```diff
@@ -16,4 +16,50 @@
     # TODO: change to `nixos.org` when ready
     domains = [ "mail-test.nixos.org" ];
   };
+
+  services.postfix.config.bounce_template_file = "${pkgs.writeText "bounce-template.cf" ''
+    failure_template = <<EOF
+    Charset: us-ascii
+    From: MAILER-DAEMON (Mail Delivery System)
+    Subject: Undelivered Mail Returned to Sender
+    Postmaster-Subject: Postmaster Copy: Undelivered Mail
+
+    This is the mail system at host $myhostname.
+
+    I'm sorry to have to inform you that your message could not
+    be delivered to one or more recipients. It's attached below.
+
+    For further assistance, please file an issue at
+    https://github.com/NixOS/infra/issues/new
```

Comment:
A downside with this is that it will likely result in people revealing the problematic email addresses publicly. @mweinelt, do you have a suggestion here?

Reply:
I think GitHub issues is indeed the best contact point for load-balancing this. Whoever submits these needs to anonymize the relevant logs as needed. If they don't do it, we still can and we can even delete the edit history.

Reply:
Sounds great. I've updated the templates to instruct people to anonymize personal email addresses in their report.